apps/examples/wifi_controller/Kconfig

```diff
@@ -3,45 +3,45 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
-config EXAMPLES_WIFIMANAGER_TEST
-	bool "Wi-Fi Manager Sample"
+config EXAMPLES_WICTL
```

Comment:
In my opinion WICTL is not easily understandable to the user, how about changing it as `EXAMPLES_WIFI_CTRL `and also changing the command name as `wifi` . 
Ex: `wifi start, wifi scan, wifi join, wifi stop` etc.